src/components/AccessibilityGuide/ProgressiveEnhancement.mdx

```diff
@@ -0,0 +1,229 @@
+issues, however, point back to DS components.
+
+## HTML Button Issue
+
+HTML `<button>`s can only work without JavaScript in the following contexts.
+
+- When used inside a `<form>` element, if it has an attribute of type="submit",
+  the button will be clickable.
+- When used inside a `<form>` element, if it has an attribute of type="reset",
+  the button will be clickable and clear out input elements.
+
+However, there are several DS components that use the `Button` component (and
+therefore the `<button>` HTML element) outside of a form to open, close, or
+perform other user actions. These components cannot fully function without
+JavaScript.
+
+## Component Audit
+
+As of v3.6.1, the DS contains 25 fully functional components, and 34 components
+that need JavaScript to work.
+
+The following (25) components are fully functional without JavaScript:
+
+- `Breadcrumbs`
+- `Card`
+- `FeaturedContent`
+- `FieldSet`
+- `Footer`
+- `Form`
+- `FormRow`
+- `FormField`
+- `HelperErrorText`
+- `Hero`
+- `HorizontalRule`
+- `Icon`
+- `Image`
+- `Label`
+- `Link`
+- `List`
+- `Logo`
+- `SimpleGrid`
+- `SkipNavigation`
+- `StatusBadge`
+- `StructuredContent`
+- `StyledList`
+- `Table`
+- `Template`
+- `Text`
+
+The following (16) use the HTML `<button>` element and are not fully functional
+without JavaScript:
+
+- `Accordion`
+- `AlphabetFilter`
+- `Banner`
+- `Button`
+- `ButtonGroup`
+- `Header`
+- `Menu`
+- `Modal`
+- `MultiSelect`
+- `MultiSelectGroup`
+- `NewsletterSignup`
+- `Notification`
+- `Pagination`
+- `SubNav`
+- `Tabs`
+- `TagSet`
+
+The following (15) components need a pattern update to work without JavaScript:
+
+- `CheckBox`
+- `CheckboxGroup`
+- `FeedbackBox`
+- `FilterBarInline`
+- `FilterBarPopup`
+- `ProgressIndicator`
+- `Radio`
+- `RadioGroup`
+- `SearchBar`
+- `Select`
+- `SkeletonLoader`
+- `Slider`
+- `TextInput`
+- `Toggle`
+- `Tooltip`
+
+The following (2) can't be fixed due to browser limitations:
+
+- `AudioPlayer`
+- `VideoPlayer`
+
+The following (1) depend on external libraries:
+
+- `DatePicker`
+
+## General Recommendations
+
+Based on the [Engineering General JavaScript standard](https://github.com/NYPL/engineering-general/blob/main/standards/javscript.md),
+a no-JavaScript fallback is preferable but not required. Our recommendation is
+to fix what we can. This audit gives us a better understanding of what components
+currently do not work without JavasSript, and which ones we have the opportunity
+to change (with time and resources).
+
+Various new pages and features being built for NYPL have JavaScript-heavy
+requirements. We recommend and encourage teams to add a noscript fallback
+message. This would be an HTML/CSS snippet (not React) that instructs patrons
+to enable Javascript to use the site.
+
+## noscript Fallback Example
+
+We recommend placing the following `noscript` snippet at the top of the
+`<body>` tag in your HTML for the global message.
```

Comment:
How do you know that this example code will work as intended?  Did you test this somewhere?

Reply:
Yes, it's been tested in Scout, Digital Collections, and Turbine.